lib/fedora_migrate/rels_ext_datastream_mover.rb

```diff
@@ -4,26 +4,32 @@ class RelsExtDatastreamMover < Mover
     RELS_EXT_DATASTREAM = "RELS-EXT".freeze
 
     def migrate
-      statements.each do |statement|
-        subject.ldp_source.graph << [subject.rdf_subject, migrate_predicate(statement.predicate), migrate_object(statement.object)]
-      end
-      subject.ldp_source.update
+      migrate_statements
+      target.ldp_source.update
       update_index
     end
 
+    def post_initialize
+      @target ||= ActiveFedora::Base.find(source.pid.split(/:/).last)
```

Comment:
Should/could this use `#id_component`?  I ask because I'm seeing multiple places in the code where you're splitting on a colon and calling last on that result.